### PR TITLE
Add checkbox component

### PR DESCRIPTION
**Pull request by HyTekCoop**
Added a new checkbox component according to the style

image

### CODE REVIEW

**Review by e11sy — COMMENTED** [review]
add checkbox example to playground please

File: codex-ui/src/vue/components/checkbox/Checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    :class="[$style.checkbox, disabled ? $style[`checkbox--disabled`] : '', checkboxValue ? $style[`checkbox--on`] : '']"
```
Comment (e11sy) [inline codex-ui/src/vue/components/checkbox/Checkbox.vue:3]:
```suggestion
    :class="[$style.checkbox, disabled && $style[`checkbox--disabled`], checkboxValue && $style[`checkbox--on`]]"
```

File: codex-ui/src/vue/components/checkbox/Checkbox.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    v-on="disabled === false ? { click: changeValue } : {}"
```
Comment (e11sy) [inline codex-ui/src/vue/components/checkbox/Checkbox.vue:4]: vue has @ as v-on shortcut e.g. @click

File: codex-ui/src/vue/components/checkbox/Checkbox.vue
Comment (elizachi) [inline codex-ui/src/vue/components/checkbox/Checkbox.vue:1]: Please add checkbox to the Playground for testing

File: codex-ui/src/vue/components/checkbox/Checkbox.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <svg
+      class="checkmark"
+      xmlns="http://www.w3.org/2000/svg"
+      width="20"
+      height="20"
+      viewBox="0 0 20 20"
+      fill="none"
+    >
+      <path
+        d="M5.99998 9.5L8.8823 12.6705C8.96165 12.7578 9.09892 12.7578 9.17828 12.6705L14.3333 7"
```
Comment (elizachi) [inline codex-ui/src/vue/components/checkbox/Checkbox.vue:15]: Seems hardcoded, is there other ways to solve it?

Comment (neSpecc) [reply codex-ui/src/vue/components/checkbox/Checkbox.vue:15]: can be imported from codex the icons package

**Pull request closed without merging**